numpyro/infer/mcmc.py

```diff
@@ -509,6 +511,8 @@ def run(self, rng_key, *args, extra_fields=(), init_params=None, **kwargs):
         """
         Run the MCMC samplers and collect samples.
 
+        See :class:`~numpyro.infer.util.Predictive` for how to use them to collect posterior predictive samples.
```

Comment:
I think `get_samples` is a better place for this. Also maybe changing `them` to `those samples`?